src/univariatestandardizeddistributions.jl

```diff
@@ -253,3 +253,78 @@ function quantile(dist::StdGED, q::Real)
     qq = 2*q-1
     return sign(qq) * (gammainvcdf(ip, 1., abs(qq)))^ip/kernelinvariants(StdGED, [p])[1]
 end
+
+################################################################################
+#Hansen's SKT-Distribution
+
+"""
+    StdSkewT{T} <: StandardizedDistribution{T}
+
+Hansen's standardized (mean zero, variance one) Skewed Student's t distribution.
+"""
+struct StdSkewT{T} <: StandardizedDistribution{T}
+    coefs::Vector{T}
+    function StdSkewT{T}(coefs::Vector) where {T}
+        length(coefs) == 2 || throw(NumParamError(2, length(coefs)))
+        new{T}(coefs)
+    end
+end
+
+"""
+    StdSkewT(v,λ)
+
+Create a standardized skewed t distribution with `v` degrees of freedom and `λ` shape parameter. `ν,λ`` can be passed
+as scalars or vectors.
+"""
+StdSkewT(ν,λ) = StdSkewT([ν λ])
+StdSkewT(ν::Integer,λ::Integer) = StdSkewT(float(ν),float(λ))
+StdSkewT(v::Vector{T},λ::Vector{T}) where {T} = StdSkewT{T}(v,λ)
```

Comment:
This doesn't work because there is no such inner constructor. Also, it probably shouldn't work, so just remove it.